test/TESTING_GUIDELINES.md

```diff
@@ -55,6 +55,17 @@ Our UI tests also run on BrowserStack. If you would like to run them on your Bro
 - Specific to Safari on BrowserStack, you need to explicitly set the `localhostUrl` value in `config.json` to
   be read as `"localhostUrl": "https://bs-local.com:8080/",` this is due to the way Safari handles `https://localhost`.
 
+### Running tests with or without specific tags
+
+- We can now run tests that contain, or do not contain certain tags in the following way...
+
+`MOCHA_GREP="@1|@2" npm run test:ui` will run tests that ONLY have the tag @1 OR @2.
```

Comment:
Just so I know I understood the usage correctly, this mean that if for example I added a tag of `@test_here` and `@test_here_also` on a couple of UI tests in the Document scenario's test description & I run `MOCHA_GREP="@test_here|@test_here_also" npm run test:ui` that would work also & it's not dependent on just a bunch of number-based tags like in the guidelines doc here?

Reply:
Yes, I just used the number based tags as an example. I'd suggest as part of this ticket, give it a try as per the acceptance criteria.